web/pandas/contribute.md

```diff
@@ -0,0 +1,12 @@
+# Contribute to pandas
+
+_pandas_ is and always will be **free**. To make de development sustainable, we need _pandas_ users, corporate
+or individual, to support the development by providing their time and money.
+
+You can find more information about current developers in the [team page](about/team.html),
+and about current sponsors in the [sponsors page](about/sponsors.html).
+Financial contributions will mainly be used to advance in the [pandas roadmap](about/roadmap.html).
+
+- If your **company or organization** is interested in helping make pandas better, please contact us at [info@numfocus.org](mailto:info@numfocus.org)
```

Comment:
Just wondering: is this the best way to contact us? (we don't have that somewhere written now I think? But for sure we also don't have our "own" email address for pandas)